docs/sidebars.js

```diff
@@ -215,5 +232,17 @@ module.exports = {
         "development/events",
       ],
     },
+    {
+      type: "category",
+      label: "Versions",
+      link: {
+        type: "generated-index",
+        title: "ZMK Versions",
+        description: "Available documentation versions",
```

Comment:
```suggestion
        description: "Below are the available documentation versions, corresponding to ZMK versions.",
```

I think this needs a little elaborating. It's awkward without being able to provide more information on ZMK versions but I guess that's fine for now.